hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/config/KafkaHeaderNameProperties.java

```diff
@@ -9,6 +9,10 @@ public class KafkaHeaderNameProperties {
 
     private String schemaId = "sid";
 
+    private String messageId = "id";
```

Comment:
Can You please move `KafkaHeaderNameParameters` from `frontend` to `common` package, and make `KafkaHeaderNameProperties` implementing [KafkaHeaderNameParameters](https://github.com/allegro/hermes/blob/ce5f351bded1d3ce15d5cc7319b1858baf47d3a3/hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/producer/kafka/KafkaHeaderNameParameters.java)? Just like for example [SchemaVersionRepositoryParameters](https://github.com/allegro/hermes/blob/ce5f351bded1d3ce15d5cc7319b1858baf47d3a3/hermes-common/src/main/java/pl/allegro/tech/hermes/common/schema/SchemaVersionRepositoryParameters.java)

Reply:
Sure, good suggestion, done :)